readthedocs/proxito/views/serve.py

```diff
@@ -673,3 +675,45 @@ def changefreqs_generator():
 
 class ServeSitemapXML(SettingsOverrideObject):
     _default_class = ServeSitemapXMLBase
+
+
+class ServeStaticFiles(CachedView, CacheTagsMixin, View):
+
+    """
+    Serve static files from the same domain the docs are being served from.
+
+    This is basically a proxy for ``STATIC_URL``.
+    """
+
+    project_cache_tag = "staticfile"
```

Comment:
```suggestion
    project_cache_tag = "staticfiles"
```

Reply:
We are using `rtd-search` and `rtd-footer` in other views, don't know about using the plural form (each file has a tag, but it could also be seen as the tag having multiple files). So, what about using `rtd-staticfile`?

Reply:
👍🏼 to the prefix `rtd-` as we are doing with the others. Also, 👍🏼 to the plural form. BTW, the method's name is called `purge_static_files` in https://github.com/readthedocs/readthedocs-ext/pull/507/files#diff-9811f516132d2a783b686e818b59a6368d126b0351b41d832d0fd0ba85493b14R119

So, IMO, `rtd-staticfiles` makes sense.

Reply:
`rtd-staticfiles` works for me. The footer & search are singular concepts. Static files is plural, so the `s` makes sense :)